### ISSUE TITLE
FFXIV launcher goes to update with says its around 44.7gigs

### ISSUE THREAD

**Issue by Catatafishes** [goatcorp/FFXIVQuickLauncher#535]
I go to update it this error occurs please help. 

System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. ---> System.UnauthorizedAccessException: Access to the path 'C:\Program Files (x86)\SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\4e9a232b' is denied.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.Directory.InternalCreateDirectory(String fullPath, String path, Object dirSecurityObj, Boolean checkHost)
   at XIVLauncher.Game.Patch.PatchManager.GetPatchFile(PatchListEntry patch) in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Game\Patch\PatchManager.cs:line 474
   at XIVLauncher.Game.Patch.PatchManager.<DownloadPatchAsync>d__33.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Game\Patch\PatchManager.cs:line 182
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Game.Patch.PatchManager.<>c__DisplayClass35_0.<<RunDownloadQueue>b__2>d.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Game\Patch\PatchManager.cs:line 335
   --- End of inner exception stack trace ---
---> (Inner Exception #0) System.UnauthorizedAccessException: Access to the path 'C:\Program Files (x86)\SquareEnix\FINAL FANTASY XIV - A Realm Reborn\game\4e9a232b' is denied.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.Directory.InternalCreateDirectory(String fullPath, String path, Object dirSecurityObj, Boolean checkHost)
   at XIVLauncher.Game.Patch.PatchManager.GetPatchFile(PatchListEntry patch) in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Game\Patch\PatchManager.cs:line 474
   at XIVLauncher.Game.Patch.PatchManager.<DownloadPatchAsync>d__33.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Game\Patch\PatchManager.cs:line 182
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at XIVLauncher.Game.Patch.PatchManager.<>c__DisplayClass35_0.<<RunDownloadQueue>b__2>d.MoveNext() in D:\a\FFXIVQuickLauncher\FFXIVQuickLauncher\src\XIVLauncher\Game\Patch\PatchManager.cs:line 335<---

Version: 6.0.6.0
Git Hash: 6.0.6-0-gac7dd80
Context: Unhandled
OS: Microsoft Windows NT 6.2.9200.0
64bit? True
DX11? True
Addons Enabled? True
Auto Login Enabled? False
Language: English
LauncherLanguage: English
Game path: C:\Program Files (x86)\SquareEnix\FINAL FANTASY XIV - A Realm Reborn

**Comment by reiichi001** [comment]
Hello!

Is this still happening?

Based on your error, this is most likely Windows permissions or your Antivirus blocking XIVLauncher/Dalamud.

**Comment by reiichi001** [comment]
Closing due to age. Please contact us in Discord or by making a new issue if you still need help.
